contracts/blockchainTest.scrypt

```diff
@@ -0,0 +1,38 @@
+import "blockchain.scrypt";
+
+
+contract BlockchainTest {
+    
+    int blockchainTarget;
+    
+    public function testBlockHeight(BlockHeader bh, MerklePaths merklePath,  bytes coinbaseTx, int height, SigHashPreimage txPreimage) {
+        require(Tx.checkPreimage(txPreimage));
+
+        // get id of previous tx
+        Sha256 prevTxid = Sha256(Util.outpoint(txPreimage)[:32]);
+
+        // validate block header
+        require(Blockchain.isBlockHeaderValid(bh, this.blockchainTarget));
+
+        // verify previous tx is in the block
+        require(Blockchain.txInBlock(prevTxid, bh, merklePath));
```

Comment:
These are not relevant to height. Put them in separate functions if u wanna test them.